packages/core/src/components/Copy/index.tsx

```diff
@@ -56,10 +56,9 @@ export default function Copy({
   };
 
   const element = children || (
-    // eslint-disable-next-line jsx-a11y/anchor-is-valid
-    <Link trackingName={trackingName} id={id}>
```

Comment:
I think this was on purpose so the copy icon could be inline with text.

Reply:
hrmm okay ill look into that!

Reply:
@milesj i added a story to test, and it's fine!

<img width="855" alt="Core : Copy - Copy inline with text   ⋅ Storybook 2020-03-27 16-12-08" src="https://user-images.githubusercontent.com/306275/77807651-bcc06080-7045-11ea-9c2f-47a6a8141dda.png">